virttest/virt_admin.py

```diff
@@ -868,5 +868,5 @@ def client_disconnect(server_name, client_id, **dargs):
     :param dargs: standardized virt-admin function API keywords.
     :return: CmdResult object
     """
-    cmd = "client_disconnect %s %s" % (server_name, client_id)
+    cmd = "client-disconnect %s %s" % (server_name, client_id)
```

Comment:
@iccaszhulili Agreed, your typo fix looks good.

But can you take this chance to fix the doctring so that API users will get benefited can be used correctly,
I meant API doctring of `def client_disconnect(server_name, client_id, **dargs)` states
```
server_name - server which the client is currently connected to
client_id - client which to disconnect
```
but as per man page mentioned in that link,
`=item B<client-disconnect> I<client> I<server>`
```
I\<client\> - Close a connection originating from I\<client\>
I\<server\> - the name of the server I<client> is currently connected to
```

Both of them are contradicting.